data.sql

```diff
@@ -13,3 +13,45 @@ VALUES
 (9, 'Boarmon', '2005-06-07', 7, 'TRUE', 20.4),
 (10, 'Blossom', '1998-10-13', 3, 'TRUE', 17),
 (11, 'Ditto', '2022-05-14', 4, 'TRUE', 22);
+
```

Comment:
- [ ] Kindly add mandatory information in your readme file because it is looking like a template. Kindly insert your data in the readme file such as Project Description, Key Feature, Future Feature, Getting Started, and License Link :+1: